sw/host/tests/rom/e2e_bootstrap_entry/src/main.rs

```diff
@@ -585,13 +630,13 @@ fn main() -> Result<()> {
         (SpiFlash::SECTOR_ERASE, "01425303"),
         // `kErrorBootstrapProgramAddress` (02425303) is defined in `error.h`.
         (SpiFlash::PAGE_PROGRAM, "02425303"),
-    ] {
-        execute_test!(test_bootstrap_shutdown, &opts, &transport, cmd, bfv);
-    }
-    execute_test!(test_bootstrap_phase1_reset, &opts, &transport);
-    execute_test!(test_bootstrap_phase1_page_program, &opts, &transport);
-    for erase_cmd in [SpiFlash::SECTOR_ERASE, SpiFlash::CHIP_ERASE] {
-        execute_test!(test_bootstrap_phase1_erase, &opts, &transport, erase_cmd);
+        ] {
+            execute_test!(test_bootstrap_shutdown, &opts, &transport, cmd, bfv);
+        }
+        execute_test!(test_bootstrap_phase1_reset, &opts, &transport);
+        execute_test!(test_bootstrap_phase1_page_program, &opts, &transport);
+        for erase_cmd in [SpiFlash::SECTOR_ERASE, SpiFlash::CHIP_ERASE] {
+            execute_test!(test_bootstrap_phase1_erase, &opts, &transport, erase_cmd);
```

Comment:
nit: can you take a look at the indentation and whitespace changes here?

Reply:
Sorry, fixed.